server.js

```diff
@@ -332,7 +340,7 @@ let routes = [
 
 let server = http.createServer(function(request, response) {
   response.writeHead(200, {
-    // 'Content-Type': 'text/plain',
+    'Content-Type': 'text/plain',
```

Comment:
My code is going to comment this out, so I'm cool with leaving this here and we can then let my PR comment it out.